metadata-db-app/src/cmr/metadata_db/services/messages.clj

```diff
@@ -128,19 +129,21 @@
           this-revision-id
           this-transaction-id))
 
-(defn pvn-equality-failure
-  [concept]
-  (format (str "The provider id [%s] and variable name [%s] combined must be "
+(defn pfn-equality-failure
+  [field-type concept]
+  (format (str "The provider id [%s] and %s [%s] combined must be "
                "unique for a given native-id [%s]. The following concept "
-               "with the same provider id, variable name and native-id was "
-               "found: [%s].")
+               "with the same provider id, %s, and native-id was found: [%s].")
           (:provider-id concept)
-          (get-in concept [:extra-fields :variable-name])
+          field-type
```

Comment:
Found the humanize think I was remembering. Pasting below.

I think just variable-name would be fine so `(name field-type)` works for me.

`cmr.common.validations.core`

```
(defn humanize-field
  "Converts a keyword to a humanized field name"
  [field]
  (when field
    (->> (str/split (csk/->kebab-case (name field)) #"-")
         (map str/capitalize)
         (str/join " "))))
```

Reply:
Perfect -- thanks!